doc/lua_api.txt

```diff
@@ -6040,6 +6040,9 @@ It can be created via `Settings(filename)`.
     * `default` is the value returned if `key` is not found.
     * Returns `nil` if `key` is not found and `default` not specified.
 * `get_np_group(key)`: returns a NoiseParams table
+* `get_flags(key)`:
+    * Returns `{flag = true/false, ...}` according to the set flags.
+    * Is currently limited to mapgen settings like `mgv5_spflags`.
```

Comment:
"Is currently limited to mapgen flags like `mg_flags` and `mgv5_spflags`."

Reply:
Well so: `Is currently limited to mapgen settings like 'mgv5_spflags' or ' mg_flags'.`?

Reply:
Hmm, what i meant was, why call these 'settings' when it is more descriptive to call them 'mapgen flags'.
But i can see the problem with my suggestion now.
How about:
"Is currently limited to the mapgen flags 'mg_flags' and 'mgv5_spflags'."
?

Reply:
"Is currently limited to mapgen flags 'mg_flags' and mapgen-specific flags like 'mgv5_spflags'"
?

Reply:
Resolved.